pkg/serviceaccount/claims.go

```diff
@@ -51,21 +53,33 @@ type kubernetes struct {
 	Pod       *ref             `json:"pod,omitempty"`
 	Secret    *ref             `json:"secret,omitempty"`
 	WarnAfter *jwt.NumericDate `json:"warnafter,omitempty"`
+
+	RequesterUserInfo userInfo `json:"requesterUserInfo,omitempty"`
```

Comment:
Should this be a chain?

Reply:
A chain of who requested the requesting user's token? I can see why that might be valuable, though I am not totally clear on the use-case for it.

For me at least, a single parent is sufficient to allow me to identify (in my case) which kubelet requested a specific pod's service account token, which allows for a greater level of assurance that a pods token is not being replayed from another host (when the host authenticates to an external service on the pods behalf).